pkg/service/repair/service_repair_integration_test.go

```diff
@@ -2029,6 +1988,47 @@ func TestServiceRepairIntegration(t *testing.T) {
 			t.Fatalf("Expected table to be fully repaired, got %d/%d", p.Tables[0].TokenRanges, p.Tables[0].Success)
 		}
 	})
+
+	t.Run("repair ignore hosts", func(t *testing.T) {
+		h := newRepairTestHelper(t, session, defaultConfig())
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
```

Comment:
After this test is executed, is starts the node again and sleeps for 5s. It might happen that the node won't be able to take part in a repair after such short startup time.

Reply:
But I will try to improve waiting mechanism instead of moving it to the back of the test list.